src/d3a/models/strategy/load_hours.py

```diff
@@ -321,7 +321,8 @@ def event_market_cycle_prices(self):
         if ConstSettings.IAASettings.MARKET_TYPE == 1:
             return
         for market in self.active_markets:
-            if self.energy_requirement_Wh[market.time_slot] > 0:
+            if market.time_slot in self.energy_requirement_Wh and \
```

Comment:
Not sure why this was needed too. I would expect that the energy requirement should always be populated with the time_slot value (for the simulation case at least, where this value is populated at the beginning of the simulation)

Reply:
Indeed. Remnant from debugging. Will remove it.